src/ORM/Rule/IsUnique.php

```diff
@@ -53,6 +53,11 @@ public function __invoke(EntityInterface $entity, array $options)
             return true;
         }
 
+        $permitMultipleNulls = true;
+        if (isset($options['permitMultipleNulls'])) {
```

Comment:
@ADmad Sounds good and reasonable to me.
@JayPHP  I'm sorry to trouble you. :bow: